mcs/class/corlib/Test/System/TimeZoneInfoTest.cs

```diff
@@ -115,6 +115,12 @@ public void GetLocal ()
 				Assert.IsTrue (true);
 			}
 
+			[Test]
+			public void LocalIdCheck ()
+			{				
+				Assert.IsTrue (TimeZoneInfo.Local.Id != "Local", "Local timezone id should not be \"Local\"");				
```

Comment:
I dug a bit deeper and I think this will fail in cases where we read the local time zone from `/etc/localtime`, like https://github.com/mono/mono/blob/2f1b50bfbf9db129d844dad8e45ec248f0c877e4/mcs/class/corlib/System/TimeZoneInfo.cs#L146-L147.

I assume that was the reason why we added the `readlink()` check to the other test.

So instead of adding this new test we can reuse the `LocalId` test and simply ifdef out the readlink on `#if MONOTOUCH || XAMMAC`